docs/docs/40-operator-guide/40-security/30-access-controls.md

```diff
@@ -110,31 +125,41 @@ As previously mentioned, _most_ access controls are managed at the project level
 by project admins, however, there are two ways in which an operator can also
 map users to `ServiceAccount` resources.
 
-#### `api.oidc.admins` / `api.oidc.viewers`
+#### `api.oidc.admins` / `api.oidc.viewers` / `api.oidc.users` / `api.projectCreators`
+
+The `api.oidc.admins`, `api.oidc.projectCreators`, `api.oidc.users`, and
+`api.oidc.viewers` configuration options of the Kargo Helm chart permit an 
+operator to map users with specific claims to _system-wide_ admin, 
+project creator, user, and viewer roles respectively. 
```

Comment:
I think we should make this a table showing the settings and a description of the permissions they include.